Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -34,36 +38,54 @@ namespace O3DE::ProjectManager
         explicit CreateGem(QWidget* parent = nullptr);
         ~CreateGem() = default;
 
+        void ClearFields();
```

Comment:
Lets move this function into `protected` scope if it's only meant to be called by this class and classes that inherit from it.